test/test_units.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import logging
+import os
+
+LOGGER = logging.getLogger(__name__)
+THIS_DIR = os.path.dirname(os.path.realpath(__file__))
+
+
+def test_units(container):
+    """Various units tests
+    Add a py file in the {image}/test/units dir and it will be automatically tested
+    """
+    short_image_name = container.image_name[container.image_name.find('/') + 1:]
```

Comment:
We're stripping `jupyter/` (or other owner part)

Reply:
I'm not sure about the point of short_image_name while writing this comment, but I assume the point is to only have the last part following all `/`. Due to this, perhaps you want to use `rfind` instead?

Example image name that I guess could be relevant in the future:

quay.io/jupyter/minimal-notebook

Reply:
```suggestion
    short_image_name = container.image_name[container.image_name.rfind('/') + 1:]
```